packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -624,7 +625,9 @@ export class ChatController {
                     triggerPayload.relevantTextDocuments = await LspController.instance.query(triggerPayload.message)
                     triggerPayload.relevantTextDocuments.forEach((doc) => {
                         getLogger().info(
-                            `amazonq: Using workspace files ${doc.relativeFilePath}, content(partial): ${doc.text?.substring(0, 200)}`
+                            `amazonq: Using workspace files ${
+                                doc.relativeFilePath
+                            }, content(partial): ${doc.text?.substring(0, 200)}`
```

Comment:
is this a formatting-only change? if our linter doesn't require it, this may indicate that your IDE (or a plugin) needs to be adjusted (or just disabled; we already have an autoformat on-commit).

Reply:
not really sure where it came from, but believe its resolved.